src/toy_models/consumption_retirement_model/exogenous_processes.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
```

Comment:
[```marginal_wealth```](https://github.com/segsell/dc-egm/blob/exog_processes/src/toy_models/consumption_retirement_model/budget_functions.py#L117) is not used anywhere. Do we need this function still?